storage-class/nfs/delete-nfs-cient-csi.sh

```diff
@@ -8,13 +8,25 @@ fi
 
 
 NS=nfs
-while true; do
-    read -p "Are you sure you want to delete nfs-csi helm chart?(Y/n) " yn
-    if [ $yn = "Y" ]
-      then
-        helm -n $NS delete nfs-csi
-        break
-      else
-        break
-    fi
-done
+
+function deleting_nfs() {
+  while true; do
+      read -p "Are you sure you want to delete nfs-csi helm chart?(Y/n) " yn
+      if [ $yn = "Y" ]
+        then
+          helm -n $NS delete nfs-csi
```

Comment:
can you cross-check with the helm release name, it should be "csi-driver-nfs"

Reply:
updated.